app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class="container mt-5 pt-5">
+  <div class="row">
+    <% @categories.each do |category| %>
+      <div class="col-6"> <!-- Bootstrap class for 2 cards per row on medium screens and up -->
+        <%= link_to category_user_transactions_path(category), class: 'category-link link' do %>
+          <div class="card mb-4">
+            <div class="row g-0">
+
+              <div class="col-md-8 justify-content-center">
+                <div class="card-body d-flex flex-column align-items-center gap-2 scale-on-hover">
+                  <h5 class="card-title link p-text"><%= category.name %></h5>
+                  <div class="col-md-4">
+                    <%= image_tag(category.icon, class: 'img-fluid rounded-start') if category.icon.present? %>
+                  </div>
+                  <p class="card-text link p-text">$<%= category.user_transactions.sum(:amount) %></p>
+                </div>
+              </div>
+
+            </div>
+          </div>
+        <% end %>
```

Comment:
- [ ] The styling in the categories page is broken, All categories are displaying different sizes, and the `add new category` button is overlapping on the categories and it also scrolls up with categories when I scroll the page. Please make sure to fix these issues.

![image](https://github.com/NoelLincoln/budget-app/assets/83803414/135d1478-e40a-44ab-8267-6d41ae26643c)

Reply:
I just noticed, thank you. Fixed